apps/tenant-management-webapp/src/app/pages/admin/services/form/definitions/formStepper.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { useState } from 'react';
+import { GoAFormStepper, GoAFormStep, GoAPages, GoAButton } from '@abgov/react-components-new';
+import { materialRenderers, materialCells } from '@jsonforms/material-renderers';
+import { JsonForms } from '@jsonforms/react';
+import { Grid, GridItem } from '@components/Grid';
+import { categorizationRendererTester } from './addEditFormDefinitionEditor';
+import FormStepperControl from './FormStepperControl';
+
+import { ReviewItem } from './style-components';
+
+interface StepperProps {
+  uiSchema: Record<string, unknown>;
+  dataSchema: Record<string, unknown>;
+  data: Record<string, unknown>;
+}
+
+declare type Status = 'complete' | 'incomplete';
+
+export const FormStepper: React.FC<StepperProps> = ({ uiSchema, data, dataSchema }) => {
+  const [step, setStep] = useState<number>(-1);
+  const [stepData, setStepData] = useState(data);
+
+  function setPage(page) {
+    if (page < 1 || page > uiSchema.elements?.length + 1) return;
+    setStep(page);
+  }
+
+  const renderers = [
+    ...materialRenderers,
+    {
+      tester: categorizationRendererTester,
+      renderer: FormStepperControl,
+    },
+  ];
+
+  return (
+    <div id="#/properties/formStepper" className="formStepper">
+      <GoAFormStepper testId="form-stepper-test" step={step} onChange={(step) => setStep(step)}>
```

Comment:
From a clean-code perspective, it would be nice to see these elements in separate files:
GoAFormStepper, GoAPages, ReviewItem, etc.